modules/govuk/templates/etc/cron.d/snapshot.erb

```diff
@@ -0,0 +1 @@
+*/10 0-4,6-23 * * * <%= @snapshot_user %> /usr/local/bin/rummager-snapshot
```

Comment:
The intention here is to run every day of the week, those ranges should be the hour setting. The time I'm trying to avoid is when jenkins runs the pruning job, as these operations share resources in elasticsearch.

I'm not sure exactly how to do the locking, because the snapshotting job doesn't block at the moment - elasticsearch gives a response straight away and continues processing the snapshot in the background. It's possible to make that request hang until it's done via a request parameter though, which means we could put a setlock around the cron if you prefer. See https://www.elastic.co/guide/en/elasticsearch/reference/1.4/modules-snapshots.html#_snapshot 

It wouldn't be foolproof, because you could always issue a snapshot/delete some other way, that would make elasticsearch go unresponsible, but maybe its still better than what we have now. I don't the think the documentation says explicitly whether the DELETE operation is supposed to block until its done, but the testing I've done so far suggests this is the case. I'm also curious how long is this going to take when we have a backlog of snapshots built up.